app/build.gradle

```
@@ -12,7 +12,7 @@ if (performSigning) {
 }
 
 android {
-    compileSdkVersion 30
+    compileSdk 31
     buildToolsVersion "30.0.3"
```

need not to update? and targetSdkVersion
```suggestion
    buildToolsVersion "31.0.0"
```